docs/document/content/user-manual/shardingsphere-jdbc/configuration/spring-boot-starter/mix.en.md

```diff
@@ -3,4 +3,82 @@ title = "Mixed Rules"
 weight = 6
 +++
 
-TODO
+The superposition between rules in mixed configuration is associated by data source name and table name.
```

Comment:
I am sorry, but actually these English statements are difficult to understand. Since you have some comments on the following configuration items, could you remove these instruction to avoid any confusion?